src/dvu/core/step.ts

```diff
@@ -3,23 +3,31 @@ import {Scope} from './scope'
 import {ValueType} from './data/data_definition'
 
 export class Step {
+  private element: Element
```

Comment:
Can we avoid having element mapped against a step? Maybe we could have a https://en.wikipedia.org/wiki/Bidirectional_map which maintains this information in the interface itself.